microcosm_flask/namespaces.py

```diff
@@ -33,22 +33,27 @@ class Namespace(object):
 
     """
 
-    def __init__(self,
-                 subject,
-                 object_=None,
-                 path=None,
-                 controller=None,
-                 version=None,
-                 enable_basic_auth=False,
-                 enable_metrics=False,
-                 identifier_type="uuid"):
+    def __init__(
+        self,
+        subject,
+        object_=None,
+        path=None,
+        controller=None,
+        version=None,
+        enable_basic_auth=False,
+        enable_metrics=False,
+        identifier_type="uuid",
+        metrics_tag_mappers=None,
```

Comment:
As with the comments [here](https://github.com/globality-corp/microcosm-metrics/pull/3), I'd prefer a callable to a mapper.